packages/serve-cli/src/cli.ts

```diff
@@ -97,12 +92,19 @@ export interface CLIContext {
 /** Inferred program options from the root command {@link cli}. */
 export type CLIGlobals = CLI extends Command<any, infer O> ? O : never;
 
-export const defaultFork = process.env.NODE_ENV === 'production' ? availableParallelism() : 1;
-
 export type CLI = typeof cli;
 
 export type AddCommand = (ctx: CLIContext, cli: CLI) => void;
 
+// we dont use `Option.default()` in the command definitions because we want the CLI options to
+// override the config file (with option defaults, config file will always be overwritten)
+export const defaultOptions = {
+  fork: process.env.NODE_ENV === 'production' ? availableParallelism() : 1,
+  host: release().toLowerCase().includes('microsoft') ? '127.0.0.1' : '0.0.0.0',
+  port: 4000,
+  polling: '10s',
```

Comment:
Isn't it pollingInterval in the actual configuration?

Reply:
These are cli options, and the flag is `--polling` in the cli.